package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "scripts": {
+    "test": "dredd-hooks-template test"
+  },
+  "private": true,
+  "devDependencies": {
+    "dredd-hooks-template": "1.1.3"
```

Comment:
Yes for the `test` command and `devDependency`. :heavy_check_mark: :heavy_check_mark: